static/src/javascripts/bootstraps/common.js

```diff
@@ -129,7 +131,9 @@ define([
             },
 
             initFastClick: function () {
-                FastClick.attach(document.body);
+                if (config.switches.iphoneConfidence && guardian.isIphone4 && guardian.inTestBucket) {
```

Comment:
This needs to be inverted, as this will prevent Androids etc from running FastClick.